File: php/uberVan.php
<?php
    require_once("./uberAdvanced.php");
    class UberVan extends UberAdvanced {
        public function __construct ($id, $license, $driver, $passengers, $typeCarAccepted, $seatsMaterial) {
            parent::__construct($id, $license, $driver, $passengers, $typeCarAccepted, $seatsMaterial);
        }

        public function setPassengers ($passengers) {
            if ($passengers >= 6) {
                $this->passengers = $passengers;
            } else {
                echo("Car Passengers Must be 6 or more");
            }
        }
    }
?>

File: python/uberVan.py
from uberAdvanced import UberAdvanced

class UberVan (UberAdvanced):
    def __init__ (self, id, license, driver, passengers, typeCarAccepted, seatsMaterialal):
        super().__init__(id, license, driver, passengers, typeCarAccepted, seatsMaterialal)

    def setPassengers (self, passengers):
        if (passengers >= 6):
            self._passengers = passengers
        else:
            print("Car Passengers must be 6 or more")